charts/covidDataExplorer/CovidCountryPicker.tsx

```diff
@@ -400,11 +400,17 @@ export class CountryPicker extends React.Component<{
                             onChange={option => {
```

Comment:
nit: I think this lambda would now be better as an @action.bound method, as it's gotten decently long. Then you could drop the runInAction.

Reply:
Good idea, done!